src/zope/component/tests/test_interface.py

```diff
@@ -13,8 +13,14 @@
 ##############################################################################
 """Tests for z.c.interface
 """
+import os
 import unittest
 
+
+DOCSTRINGS_REMOVED = False
+if str(os.environ.get('PYTHONOPTIMIZE')) == '2':
```

Comment:
```suggestion
if os.environ.get('PYTHONOPTIMIZE') == '2':
```

Reply:
I was just unsure if the values in `os.environ` are always strings to begin with.